services/docker-compose.devel.yml

```diff
@@ -107,19 +96,18 @@ services:
     volumes:
       - ./static-webserver/client/source-output:/static-content
     environment:
-      - SERVER_LOG_LEVEL=info
+      SERVER_LOGLEVEL : info
 
   webserver:
-    volumes:
-      &webserver_volumes_devel
+
+    volumes: &webserver_volumes_devel
       - ./web/server:/devel/services/web/server
       - ../packages:/devel/packages
-    environment:
-      &webserver_environment_devel
-      WEBSERVER_REMOTE_DEBUGGING_PORT: 3000
-      SC_BOOT_MODE: debug
-      WEBSERVER_LOGLEVEL: ${LOG_LEVEL:-DEBUG}
+    environment: &webserver_environment_devel
+      <<: *common-environment
+      WEBSERVER_LOGLEVEL: ${LOGLEVEL:-DEBUG}
```

Comment:
2 things:
- why LOGLEVEL instead of LOG_LEVEL?
- why do you keep WEBSERVER_LOGLEVEL with a defaut here vs all the others ? (e.g. you removed AUTOCALING_LOG_LEVEL for example)

Reply:
this is a mistake. good catch